src/storage/__tests__/migrations-test.js

```diff
@@ -102,13 +102,18 @@ describe('migrations', () => {
     },
   };
 
-  // What `base` becomes after all migrations.
-  const endBase = {
+  // What `base` becomes after migrations up through 62.
+  const base62 = {
     ...base52,
-    migrations: { version: 62 },
```

Comment:
This `migrations` line should stay — it makes the description of this "base" value accurate, and it's what we do with the other "baseN" values.